scalafix/input/src/main/scala/fix/ZIOSpec.scala

```diff
@@ -1638,19 +1638,19 @@ object ZIOSpec extends DefaultRunnableSpec {
         } yield assert(res._1)(equalTo(List(0, 2, 4, 6, 8))) && assert(res._2)(equalTo(List(1, 3, 5, 7, 9)))
       }
     ),
-    suite("provideCustomServices")(
```

Comment:
I think in the general case, it would be a mistake to try and rename user test names, so it's more accurate for our in Specs to have an old name and _keep_ the old name in the output.

Reply:
💯